presentation/src/main/java/com/android254/presentation/home/components/HomeSpeakersSection.kt

```diff
@@ -45,7 +46,8 @@ fun HomeSpeakersSection(
         LazyRow(
             modifier = Modifier
                 .testTag("speakersRow")
-                .padding(top = 24.dp)
+                .padding(top = 24.dp),
+            horizontalArrangement = Arrangement.spacedBy(16.dp)
```

Comment:
Minor: We can have different spacing values in the Design System. ie. SpacingLarge = 16.dp

Reply:
Yes we can work on this after Droidcon KE